src/pf-modal/index.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+    <title>pf-tooltip example</title>
```

Comment:
**Test page of pf-modal** -- https://rawgit.com/dabeng/patternfly-webcomponents/modal-dist/src/pf-modal/index.html